ДЗ 9

dp140482 opened the pull request:
"Заглушка". Выполнены задания 1, 2. Подготовлены к отправке в Realtime Database сообщения чата.

dp140482 committed:
ДЗ 7 пп. 1, 2

dp140482 committed:
ДЗ 7 п. 3

Контейнеры и презентационные компоненты

dp140482 committed:
ДЗ 7 п. 4

Redux-persist с доработкой добавления начальных чатов во избежание их дублирования

dp140482 committed:
ДЗ 8

dp140482 committed:
Подключение Firebase и начало настройки

PublicRoute и PrivateRoute

dp140482 committed:
PrivateRoute

dp140482 committed:
Sign up / Sign in / Sign out

dp140482 committed:
Для отправки сообщений в БД

SokolAlexander reviewed (changes requested):
То что есть - гуд) но пока есть не все)

dp140482 committed:
ДЗ 9

Формально задания 1 – 3 выполнены. Добавление сообщений идёт через useEffect в компонентах. Store отключен.
